### ISSUE TITLE
Add automated lighthouse tests

### ISSUE THREAD

**Issue by BeniRupp** [BeniRupp/benjaminrupp.de-2024#2]
Automated lighthouse tests should be added to track performance, accessibility and SEO metrics.
